spec/ruby/core/dir/for_fd_spec.rb

```diff
@@ -75,3 +76,4 @@
     end
   end
 end
+end
```

Comment:
Leads to

```
  An exception occurred during: after :all ERROR
  Errno::EBADF: Bad file descriptor - closedir
  /Users/runner/work/ruby/ruby/src/spec/ruby/core/dir/fixtures/common.rb:133:in 'DirSpecs.delete_mock_dirs'
  /Users/runner/work/ruby/ruby/src/spec/ruby/core/dir/for_fd_spec.rb:12:in 'block (4 levels) in <top (required)>'
  /Users/runner/work/ruby/ruby/src/spec/ruby/core/dir/for_fd_spec.rb:6:in 'block (2 levels) in <top (required)>'
  /Users/runner/work/ruby/ruby/src/spec/ruby/core/dir/for_fd_spec.rb:5:in 'block in <top (required)>'
  /Users/runner/work/ruby/ruby/src/spec/ruby/core/dir/for_fd_spec.rb:4:in '<top (required)>'
  ```
https://github.com/ruby/ruby/actions/runs/12638209495/job/35213999477?pr=12517